book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+However, this may be possible if the risk prediction corresponds to a particular model form.
+This is discussed further in @sec-survtsk-PI.
 
-Predicting rankings is the primary form of the *survival ranking task*, defined by predicting a continuous value, $g: \calX \rightarrow \calR$ where $\calR \subseteq \Reals$.
+More common is transformation from a distribution to a risk.
+Theoretically the simplest way to do so would be to take the mean or median of the distribution, however as will be discussed in detail in @sec-survtsk-time this is practically difficult.
+Instead, a stable approach is to use the 'ensemble' or 'expected mortality' to create a measure of risk [@Ishwaran2008].
+The expected mortality is defined by
+
+$$
+\sum_t -\log(\hatS_i(t)) = \sum_t \hatH_i(t)
+$$
+
+and is closely related to the calibration measure defined in @sec-alpha.
+This represents the expected number of events for individuals with similar characteristics to $i$.
+This is a measure of risk as a larger value indicates that among individuals with a similar profile, there is expected to be a larger number of events and therefore have greater risk than those with smaller ensemble mortality.
+For example, say for an individual, $i$, we have: $(t, \hatS_i(t)) = (0, 1), (1, 0.8), (2, 0.4), (3, 0.15)$ then, $(t, \hatH_i(t)) = (0, 0), (1, 0.10), (2, 0.40), (3, 0.82)$ and their relative risk prediction is $\sum_t \hatH_i(t) = 0 + 0.10 + 0.4 + 0.82 = 1.32$.
 
 ## Predicting Survival Times {#sec-survtsk-time}
 
-Predicting a time to event is the problem of predicting the expectation $\hat{y}=\EE(Y|\xx)$.
-A time-to-event prediction is a special case of a ranking prediction as an individual with a longer survival time will have a lower overall risk: if $\hat{y}_i,\hat{y}_j$ and $\hat{r}_i,\hat{r}_j$ are survival time and ranking predictions for subjects $i$ and $j$ respectively, then $\hat{y}_i > \hat{y}_j \Rightarrow \hat{r}_i < \hat{r}_j$.
+Predicting a time-to-event is the problem of understanding exactly when an individual will experience an event.
+Mathematically, the problem is the task of estimating $g: \calX \rightarrow \PReals$, that is, predicting a single value over $[0,\infty]$.
 
 For practical purposes, the expected time-to-event would be the ideal prediction type as it is easy to interpret and communicate.
-However, this type of prediction is rare for multiple reasons. For one, an usuall loss based on $f(y_i)$ or some difference of true and predicted value, $y_i-\hat{y}_i$ is not, suitable for censored data, as $y_i$ is not observed for some observations, so direct estimation/prediction of $\hat{y}_i = E(Y|\xx_i)$ requires some imputation of censored observations (and evaluation on new data can also only be done on observed or imputed values).
+However, such predictions are uncommon as the training data is never fully observed and hence training models is inherently a probabilistic process, with uncertainty being captured from the moment data is censored.
```

Comment:
a bit confusing here. Do you mean not fully observed due to censoring or because the training data is a sample from an assumed population. You don't need censoring for uncertainty. If its a sample from population, you still have uncertainty, as the observed event times are just samples from a distribution.

Reply:
In practice not possible due to censoring because the ground truth is unknown.